Tests/test_PDB_MMCIFParser.py

```diff
@@ -276,12 +326,12 @@ def test_conversion(self):
         self.assertEqual(len(pdb_struct), len(cif_struct))
 
         pdb_atom_names = [a.name for a in pdb_struct.get_atoms()]
-        cif_atom_names = [a.name for a in pdb_struct.get_atoms()]
+        cif_atom_names = [a.name for a in cif_struct.get_atoms()]
```

Comment:
I know that it is unrelated to the issue, but it obviously looks as typo error

Reply:
It does look like a typo - I'd happily merge this fix (and the matching change later in ``testModels``) as a separate PR while we wait for a review of the main changes proposed on #1775.